eras/shelley/impl/src/Cardano/Ledger/Shelley/Rules/Epoch.hs

```diff
@@ -202,18 +202,11 @@ epochTransition = do
       TRC (ShelleyPoolreapEnv vstate, PoolreapState utxoSt acnt dstate pstate', e)
 
   let adjustedCertState = CertState vstate pstate'' dstate'
-      epochState' =
-        EpochState
-          acnt'
-          (ls {lsUTxOState = utxoSt', lsCertState = adjustedCertState})
-          ss'
-          nm
-          & prevPParamsEpochStateL .~ pr
-          & curPParamsEpochStateL .~ pp
```

Comment:
I always hated these last two lines! They seem overly defensive but also i felt uneasy to remove them (because what if something gets changed and we forget to set them).
But this now can't happen, good riddance!